speid/views.py

```diff
@@ -17,27 +16,29 @@ def health_check():
 
 @app.route('/orden_events', methods=['POST'])
 def create_orden_events():
-
     if "id" not in request.json or int(request.json["id"]) <= 0:
         return make_response(jsonify(request.json), 400)
 
     request_id = request.json['id']
     transaction = db.session.query(Transaction).\
         filter_by(orden_id=request_id).one()
-    if transaction is None:
-        raise OrderNotFoundException(f'Order Id: {request_id}')
-    else:
-        transaction.estado = Estado.get_state_from_stp(request.json['Estado'])
+    if transaction is not None:
```

Comment:
`.one()` manda un `NotFoundException` cuando no encuentra un resultado, por lo que no hay un caso en el que `transaction` es `None`. 
¿Qué deberíamos hacer si recibimos una `orden_id` que no tengamos registrada? Ahora mismo solo va a levantar una excepción que STP recibe pero no sabe interpretar así que va a seguir intentando enviarla.

Reply:
Vi que antes mandaba raise OrderNotFoundException(f'Order Id: {request_id}'), es esa la excepcioón de que stp debe recibir en caso de que no exista la orden o realmente stp no lo tiene contemplado?

Reply:
`OrderNotFoundException` se creo para cuando Sentry estuviera integrado. Por eso solo levanta una excepción. STP solo espera recibir un `got it!!`, cualquier otro mensaje lo tomará como error y lo volverá a intentar enviar.

Reply:
Como no es parte de Celery esa parte la puedo integrar a Sentry sin problema, cómo ves? Aunque seguiría el mismo problema con stp que seguiría reintentarlo